README.md

```diff
@@ -38,6 +34,17 @@ in the same bitness as your Node.js installation.
    * Either install Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools`
    * Or manually install [Python 2.7](http://www.python.org/download/) and [Visual Studio](https://www.visualstudio.com/downloads/) (or modify an existing installation) and select Common Tools for Visual C++ during setup.
 
+## Enabling the SDK
+To import the Yoti SDK inside your project, you can use your favourite dependency management system.
+If you are using NPM, you need to add the following dependency:
```

Comment:
Thoughts on this? It looks odd to me and doesn't read well as it currently does. Suggested change:

```
If you are using NPM, you can use `npm install -S -E yoti-node-sdk` to set the Yoti SDK as a dependancy.

Your package.json file will then be updated to include:

```json
"dependencies": {
     "yoti-node-sdk" : "1.0.2"
 }```

 ```